### PR TITLE
Adding default timeout for adbExec and refactoring tests for CI

### PR DESCRIPTION
**Pull request by moizjv**
- Adding default timeout to adbExec to 20 sec 
- Fixing helper methods
- Setting babel version to exact 5.5.5
- Setting up CI :) 
- Refactoring tests for CI.
- @jlipps @sebv please review

### CODE REVIEW

File: lib/tools/system-calls.js [hunk 1]
```diff
@@ -151,6 +151,10 @@ systemCallMethods.adbExec = async function (cmd, opts = {}) {
   if (!cmd) {
     throw new Error("You need to pass in a command to adbExec()");
   }
+  // setting default timeout for each command to prevent infinite wait.
```
Comment (sebv) [inline lib/tools/system-calls.js]: opts.timeout = opts.timeout || 20000;

**Comment by sebv** [comment]
One comment, otherwise lgtm.

**Comment by moizjv** [comment]
@sebv fixed. Thank for pointing that out.

File: lib/helpers.js [hunk 2]
```diff
@@ -83,11 +83,21 @@ function getIMEListFromOutput (stdout) {
 }
 
 function getJavaForOs () {
-  let java = path.resolve(process.env.JAVA_HOME, 'bin', 'java');
-  if (system.isWindows()) {
-    java = java + '.exe';
+  if (process.env.JAVA_HOME) {
```
Comment (jlipps) [inline lib/helpers.js]: why doesn't this just use `getJavaHome` instead of duplicating the logic?

**Comment by jlipps** [comment]
one comment otherwise lgtm

**Comment by moizjv** [comment]
@jlipps thanks for pointing it out. Fixed.

**Pull request merged**